receiver/k8sclusterreceiver/collection/collector.go

```diff
@@ -40,35 +40,18 @@ const (
 	containerType = "container"
 
 	// Resource labels keys for UID.
-	k8sKeyPodUID                   = "k8s.pod.uid"
 	k8sKeyNodeUID                  = "k8s.node.uid"
-	k8sKeyCronJobUID               = "k8s.cronjob.uid"
-	k8sKeyDeploymentUID            = "k8s.deployment.uid"
-	k8sKeyJobUID                   = "k8s.job.uid"
 	k8sKeyNamespaceUID             = "k8s.namespace.uid"
```

Comment:
What about others? Are they not defined in core?

Reply:
The others are not yet defined in the semantic convention, hence not in core either. Here's a list of what we currently have: https://github.com/open-telemetry/opentelemetry-specification/blob/master/specification/resource/semantic_conventions/k8s.md